java/connector-node/risingwave-sink-cassandra/pom.xml

```diff
@@ -27,18 +27,6 @@
             <groupId>com.risingwave</groupId>
             <artifactId>connector-api</artifactId>
         </dependency>
-        <dependency>
```

Comment:
As a library, we should not depend on the implementation but only the `slf4j-api`.